$MirOS: ports/www/php/patches/patch-main_php_sprintf_c,v 1.1.7.1 2005/03/18 16:00:00 tg Exp $
--- main/php_sprintf.c.orig	Tue Jul 26 08:38:03 2005
+++ main/php_sprintf.c	Wed Jan 25 17:54:43 2006
@@ -35,7 +35,7 @@ php_sprintf (char*s, const char* format,
 
   va_start (args, format);
   s[0] = '\0';
-  ret = vsprintf (s, format, args);
+  ret = (char *)vsprintf (s, format, args);
   va_end (args);
   if (!ret)
     return -1;
